lms/djangoapps/program_enrollments/models.py

```diff
@@ -136,6 +137,18 @@ class Meta(object):
     def is_active(self):
         return self.status == ProgramCourseEnrollmentStatuses.ACTIVE
 
+    @property
+    def course_staff(self):
+        role_assignment_exists = CourseAccessRoleAssignment.objects.filter(
```

Comment:
I think you could follow the foreign key relationship [backward](https://docs.djangoproject.com/en/2.2/topics/db/queries/#following-relationships-backward) from this model using the `courseaccessroleassignment_set` manager, which would automatically get instances of `CourseAccessRoleAssignment` that have a foreign-key relationship to a particular instance of `ProgramCourseEnrollment`.

I think this would work and might make a little better use of the foreign-key relationship.
```
role_assignment_exists = courseaccessroleassignment_set.filter(role=ProgramCourseEnrollmentRoles.COURSE_STAFF).exists()
```

What do you think?